platform/javascript/os_javascript.cpp

```diff
@@ -134,6 +134,14 @@ bool OS_JavaScript::_check_internal_feature_support(const String &p_feature) {
 	if (p_feature == "JavaScript")
 		return true;
 #endif
+#ifndef NO_THREADS
+	if (p_feature == "threads")
```

Comment:
These should use braces. (Same with the existing `JavaScript` check, it was likely missed by `clang-tidy` as this file isn't compiled on Linux.)